update_distfiles

```diff
@@ -26,7 +26,7 @@ eval set -- "${FLAGS_ARGV}"
 switch_to_strict_mode
 
 if [[ $# -eq 0 ]]; then
-    eval set -- portage-stable coreos-overlay
+    eval set -- gentoo-subset coreos
```

Comment:
As above.

Reply:
> I don't think this script is currently used, but I'll fix it anyway.

Can you check that: https://github.com/search?q=update_distfiles+org%3Aflatcar&type=code

(i'm referring to the usage within `flatcar-linux-infra`).

Reply:
The script seems to upload to CoreOS servers, but perhaps I'm misunderstanding it.

Reply:
That script runs on the host behind https://mirror.release.flatcar-linux.net/ and syncs our distfile mirror. This is part of our infra, I just realized that it would also stop working with these changes :/

Reply:
Okay. I've opened a PR for that.